try_ABCpy.py

```diff
@@ -0,0 +1,41 @@
+import sbibm
```

Comment:
This file should go out. Either we delete it entirely, or, nicer solution: Parts of it could go into a test case, as parts of `sbibm/tests/algorithms/test_abcpy.py`. The next question would then be what exactly we want to test: E.g. either we have some expectations about how rejection ABC should perform on a subset of tasks or, alternatively, we just check for correctness of returned outputs in terms of shapes

Reply:
I've created a test using the code which was in that file.

Reply:
There I am testing the shape of the output and the actual number of performed simulations using the the possible ways to parametrize the Rejection ABC